Dockerfile

```diff
@@ -9,7 +9,7 @@ FROM registry.access.redhat.com/ubi9/openjdk-17:1.15 AS builder
 USER root
 WORKDIR /opt/kroxylicious
 COPY . .
-RUN ./mvnw -B clean verify -Pdist,withAdditionalFilters -Dquick -pl :kroxylicious-bom,:kroxylicious-app,:kroxylicious-multitenant,:kroxylicious-record-validation -am
+RUN ./mvnw -B clean verify -Pdist,withAdditionalFilters -Dquick -am
```

Comment:
This causes the docker build to fail for me with:

```
#0 76.92 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:testCompile (default-testCompile) on project kroxylicious-integration-tests: Fatal error compiling: error: release version 21 not supported -> [Help 1]
```

because we build on jdk 17. The ubi9 jdk 21 build image is available now:

```
-FROM registry.access.redhat.com/ubi9/openjdk-17:1.15 AS builder
+FROM registry.access.redhat.com/ubi9/openjdk-21:1.17 AS builder
```
which looks to build it, but the image build is now failing for me with something that looks unrelated while installing the microdnf dependencies

```
#0 104.4 error: Cannot download Packages/t/tzdata-2023c-1.el9.noarch.rpm: All mirrors were tried; Last error: Interrupted by header callback: Inconsistent server data, reported file Content-Length: 864432, repository metadata states file length: 864888 (please report to repository maintainer)
```

Reply:
I think our reinstall of the tzdata could be removed, I believe it was cargo-culted from Strimzi

https://github.com/jboss-container-images/amqstreams-1-openshift-image/pull/391/files

```
# The ubi8-minimal image intentionally removes /usr/share/zoneinfo when installing the `tzdata` RPM package to save space.
# However, since /usr/share/zoneinfo is needed by the kafka_exporter and cekit doesn't support RPM reinstall at this time,
# we must reinstall the `tzdata` RPM package manually.
```

Reply:
[upstream issue](https://issues.redhat.com/browse/RHEL-17428) created, looks like a problem to me

Reply:
"This causes the docker build to fail for me with" - I fixed this simply by removing the module from the reactor on the command line.   The real issue is that modules such as system-tests, integration-tests and performance-tests have no business being built on the quick profile.  Let's fix that separately.